src/exampleTypes/ResultTable/index.js

```diff
@@ -123,7 +124,7 @@ let ResultTable = ({
             }}
           />
         </Table>
-        {node.pageSize > 0 && (
+        {!hideFooter && node.pageSize > 0 && (
```

Comment:
What use case do we have for hiding the footer? If I recall, it only shows up if there are multiple pages

Reply:
Yeah it shows the footer for the results count